sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -183,6 +203,35 @@ func (s *secretDispatcher) processDockerConfigEvent(secret *v1.Secret, action ce
 		_ = utils.Should(errors.New("only Docker Config secrets are allowed"))
 		return nil
 	}
+	return &dockerConfig
+}
+
+func idSetFromSecret(secret *v1.Secret) (set.StringSet, error) {
+	if secret == nil {
+		return nil, nil
+	}
+	dockerConfigPtr := getDockerConfigFromSecret(secret)
+	if dockerConfigPtr == nil {
+		return nil, nil
+	}
+	dockerConfig := *dockerConfigPtr
+	registrySet := set.NewStringSet()
```

Comment:
Should be image integration.
```suggestion
	imageIntegrationIDSet := set.NewStringSet()
```